mutiny-wasm/src/lib.rs

```diff
@@ -910,6 +911,18 @@ impl MutinyWallet {
         Ok(self.inner.lnurl_withdraw(&lnurl, amount_sats).await?)
     }
 
+    /// Calls upon a Cash mint and melts the token from it.
+    #[wasm_bindgen]
+    pub async fn melt_cashu_token(
+        &self,
+        maybe_token: String,
+    ) -> Result<JsValue /* Vec<MutinyInvoice> */, MutinyJsError> {
+        let token = TokenV3::deserialize(maybe_token)?;
+        Ok(JsValue::from_serde(
+            &self.inner.melt_cashu_token(token).await?,
```

Comment:
We need to convert this into mutiny-wasm's version of `MutinyInvoice`

Have to do something like this.

```rust
let result = self.inner.melt_cashu_token(token).await?;
let invoices: Vec<MutinyInvoice> = result.into_iter().map(|i| i.into()).collect();
Ok(JsValue::from_serde(&invoices)?)
```